src/layouts/Base.astro

```diff
@@ -29,11 +34,26 @@ const jsonLDSchema = JSON.stringify(
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/ico" href="/favicon.ico" />
 		<meta name="generator" content={Astro.generator} />
+		<meta
+			name="description"
+			content="Gain practical experience by working remotely on real world projects with other early-career developers."
+		/>
+		<meta property="og:title" content="The Collab Lab" />
+		<meta
+			property="og:description"
+			content="Gain practical experience by working remotely on real world projects with other early-career developers."
+		/>
+		<meta
+			property="og:image"
+			content={`${canonicalURL.href}img/participate-developers-collage.webp`}
+		/>
+		<meta property="og:url" content={canonicalURL.href} />
 		<title>
 			{title === 'The Collab Lab' ? title : `${title} | The Collab Lab`}
 		</title>
+		<link rel="canonical" href={canonicalURL.href} />
+		<link rel="icon" type="image/ico" href="/favicon.ico" />
```

Comment:
This appears to `404`

https://the-collab-lab.codes/favicon.ico

Can we add an icon or point this at an existing asset?

Reply:
Oops! This is a bad href. I meant to put `/img` in front

Reply:
Wait, no, this is correct! It 404s because the file is in the wrong place on the deployed site. This PR moves it. You see the icon if you open the deploy preview on this PR or if you go to `https://deploy-preview-29--the-collab-lab.netlify.app/favoicon.ico`

Reply:
Ah, that makes sense!